Makefile

```diff
@@ -115,11 +118,12 @@ start: install
 	$(INSTALL_DIR)/bin/python manage.py runserver $(PORT) --settings=tests.settings
 
 # Lint the project
-lint: clean_working_directory
+lint: clean_working_directory $(INSTALL_DIR)/bin/flake8
 	$(call header,"Linting code")
-	@find . -type f -name '*.py' -not -path '$(INSTALL_DIR)/*' -not -path './docs/*' -not -path './build/*' | xargs flake8
+	@find . -type f -name '*.py' -not -path '$(INSTALL_DIR)/*' -not -path './docs/*' -not -path './build/*' | xargs $(INSTALL_DIR)/bin/flake8
+	@. $(INSTALL_DIR)/bin/activate; isort --check-only --recursive $(APP_NAME) tests
```

Comment:
No issues having them in virtualenv, I'm just saying that we already have them in virtualenv without doing any extra steps, because both flake8 and isort are in our requirements.txt file.

As long as the `$(INSTALL_DIR)` step has been run, both flake8 and isort will be accessible, either by calling out to `$(INSTAL_DIR)/bin/flake8` or by calling `$(INSTALL_DIR)/bin/activate` and then using `flake8` without a path perfix (activate will set up your path to point first to the virtualenv)